content/questions/actions/question-092.md

```diff
@@ -0,0 +1,25 @@
+---
+archetype: "questions"
+title: "Question 092"
+question: "In a GitHub Actions workflow, how do you specify a specific version of Node.js to use in a job?"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs#specifying-the-nodejs-version
+1. [x] 
+```yaml
+  uses: actions/setup-node@v4 with node-version
```

Comment:
Let's use proper github actions syntax. I want the set of questions on ghcertified to be crystal clear and not tricky or confusing, just checking the knowledge

```suggestion
  uses: actions/setup-node@v4
  with:
    node-version: 20
```